shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -817,7 +817,6 @@ - (void)dealloc {
 
 - (void)applicationBecameActive:(NSNotification*)notification {
   TRACE_EVENT0("flutter", "applicationBecameActive");
-  self.view.accessibilityElementsHidden = NO;
```

Comment:
I am confused at the change, it seems like the a11y was hidden when the view is not active prior to this change. and this change makes it so it does not hide the a11y when the view is not active?

Reply:
This code was a workaround to fix the same issue that this PR is trying to fix. Since we found the root cause and potentially a fix with this PR; we can remove the old workaround.
The workaround was introduced in https://github.com/flutter/engine/commit/0be030362e2d14b522ddc29ee5e6e480feed5d09